yb-client-cdc-stream-wrapper/src/main/java/Main.java

```diff
@@ -0,0 +1,54 @@
+import com.google.common.net.HostAndPort;
+import org.yb.client.AsyncYBClient;
+import org.yb.client.ListTablesResponse;
+import org.yb.client.YBClient;
+import org.yb.client.YBTable;
+import org.yb.master.MasterDdlOuterClass.ListTablesResponsePB.TableInfo;
+import java.util.Set;   
+import java.util.HashSet;
+
+import java.util.Objects;
+
+
+public class Main {
+    public static void main(String[] args) throws Exception {
+        CmdLineParams parameters = CmdLineParams.createFromArgs(args);
+        AsyncYBClient asyncClient = new AsyncYBClient.AsyncYBClientBuilder(parameters.masterAddresses)
+                .sslCertFile(parameters.certFilePath) // TODO: support sslClientCertFiles(clientCert, clientKey)
+                .build();
+
+        YBClient client = new YBClient(asyncClient);
+
+        if (parameters.deleteStreamId != null) {
+            Set<String> streamIds = new HashSet<String>();
+            streamIds.add(parameters.deleteStreamId);
+            client.deleteCDCStream(streamIds, false, true);
+        }  else if (parameters.toCreate) {
+            YBTable table = null;
+            try {
+                ListTablesResponse resp = client.getTablesList();
+                for (TableInfo tableInfo : resp.getTableInfoList()) {
+                    String qualifiedTableName = String.format("%s.%s", tableInfo.getPgschemaName(), tableInfo.getName());
+                    if (Objects.equals(qualifiedTableName, parameters.tableName)) {
+                        table =  client.openTableByUUID(tableInfo.getId().toStringUtf8());
+                    }
+                }
+            } catch (Exception ex) {
+                throw ex;
+            }
+            if (table == null) {
+                throw new RuntimeException("No table found with the specified name");
+            }
+            String streamID = client.createCDCStream(table, parameters.dbName, "PROTO", "IMPLICIT", "ALL").getStreamId();
```

Comment:
Pls change this to EXPLICIT. Had a chat with Suranjan where he explained that IMPLICIT checkpointing could lead to data loss. EXPLICIT checkpointing is where the cdc waits for the client to tell it that it has processed a record before checkpointing.